### PR TITLE
Fix bug 1457975: Promote Enterprise beta

### PR DESCRIPTION
**Pull request by stephaniehobson**
## Description

Fix bug 1458040
- add Salesforce lead capture form and thank you page

Fix bug 1458036
- update copy and add link to lead capture form behind a switch
- update laptop image

Fix bug 1458037
- update copy on organizations page to direct people to /enterprise behind a switch
- update logo

## Localization

English only for now.

## Issue / Bugzilla link

[Bug 1458040 - Implement Salesforce Lead Capture Form For ESR60](https://bugzilla.mozilla.org/show_bug.cgi?id=1458040)
[Bug 1458036 - Copy & Design Changes To /firefox/enterprise/ For ESR60](https://bugzilla.mozilla.org/show_bug.cgi?id=1458036)
[Bug 1458037 - Copy & Design Changes To /firefox/organizations/ For ESR60](https://bugzilla.mozilla.org/show_bug.cgi?id=1458037)

## Testing
- Buttons work
- Link works
- Pages added correctly
- Logo updated on /organizations
- High and low res assets displaying
- Form and thank you page will not be indexed

### CODE REVIEW

**Comment by stephaniehobson** [comment]
@jpetto Could you please look at this first thing? It must go out Tuesday.

We don't have sign off yet but we can't delay the review any longer. If more changes come in I'll figure out how to handle them based on what they are.

**Comment by stephaniehobson** [comment]
@rraue They asked us to make the form page not indexable. I added `rel=nofollow` to the links to the page and `<meta name="robots" content="noindex,follow">` to each page. Did I do that right?

**Comment by rraue** [comment]
@stephaniehobson The links should be follow, as crawlers follow anyways but dont count the linkpower. meta noindex, follow is enought and the way to got.

File: media/css/firefox/organizations.less [hunk 1]
```diff
@@ -4,6 +4,9 @@
 
 @import "../sandstone/lib.less";
```
Comment (jpetto) [inline media/css/firefox/organizations.less]: This issue exists in prod currently, but in IE 10 (and maybe others?) the FAQ content bleeds off the edge of the page, making quite a bit of text unreachable/unreadable.

If you don't have time to address this here, we should file a bug. (Happy to file it if you're focused on getting this page out - just let me know.)

File: bedrock/firefox/templates/firefox/enterprise/signup-thanks.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+{% block article %}
+  <h1 class="thanks-head">Thank you for signing up for Firefox Quantum for Enterprise.</h1>
+
+  <h2 class="thanks-subhead"><span class="highlight">Get Started</span></h2>
+  <p><a href="https://support.mozilla.org/en-US/products/firefox-enterprise" rel="external">Here is where you'll find documentation and support</a> regarding how to configure and deploy Firefox Quantum in your organization. You may choose to deploy the standard Firefox Rapid Release (RR), which receives performance and feature improvements every six weeks, or the Extended Support Release (ESR), which receives one major update per year. Both Firefox RR and ESR receive critical security fixes as soon as possible.</p>
+  <p>For your convenience, we’ve also sent you an email with links to this documentation.</p>
+
+  <h2 class="thanks-subhead"><span class="highlight">Give us your feedback</span></h2>
+  <p>We would love to hear from you regarding any issues you might encounter in your testing, or if you have feature requests. We encourage you to <a href="https://www.surveygizmo.com/s3/4327112/Firefox-Enterprise-users-roadmap-contribution" rel="external">complete this survey here</a>.</p>
```
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply bedrock/firefox/templates/firefox/enterprise/signup-thanks.html]: Agree 💯 Just trying to get it out the door.

**Comment by jpetto** [comment]
For posterity, tested in IE 9-11, Safari, Chrome, & Fx.

File: bedrock/firefox/templates/firefox/enterprise/signup.html [hunk 3]
```diff
@@ -0,0 +1,402 @@
...
+  <div class="content">
+    {{ high_res_img('logos/firefox/logo-quantum.png', {'alt': 'Firefox', 'width': '50', 'height': '50', 'class': 'enterprise-logo'}) }}
+    <span class="enterprise-name"><strong>Firefox Quantum</strong> for Enterprise</span>
+</div>
+</header>
+{% endblock %}
+
+{% block article %}
+
+  <form action="https://test.salesforce.com/servlet/servlet.WebToLead?encoding=UTF-8" method="POST" class="enterprise-form">
```
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply bedrock/firefox/templates/firefox/enterprise/signup.html]: I'm expecting the final URL at noon. Not sure why I can't have it before then.

File: bedrock/firefox/templates/firefox/enterprise/signup-thanks.html [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+<header class="enterprise-head">
+  <div class="content">
+    {{ high_res_img('logos/firefox/logo-quantum.png', {'alt': 'Firefox', 'width': '50', 'height': '50', 'class': 'enterprise-logo'}) }}
+    <span class="enterprise-name"><strong>Firefox Quantum</strong> for Enterprise</span>
+</div>
+</header>
+{% endblock %}
+
+{% block article %}
+  <h1 class="thanks-head">Thank you for signing up for Firefox Quantum for Enterprise.</h1>
```
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply bedrock/firefox/templates/firefox/enterprise/signup-thanks.html]: The survey is English only. As far as I know there are no plans to localize. I think the only stuff that was wrapped was meta data I copy-pasted. I've updated those strings to be page specific and not wrapped.

File: media/css/firefox/organizations.less (hunk 1 above)
Comment (stephaniehobson) [reply media/css/firefox/organizations.less]: Turns out I can test in IE8, 9, 11, and Edge but not 10.

I was able to replicate this in Firefox by falling back to the prefixed version so I suspect the problem was a conflict between the current spec and the 2012 syntax. I've attempted a quick fix for this by removing the prefixed flexbox declarations.

File: bedrock/firefox/templates/firefox/enterprise/signup.html [hunk 5]
```diff
@@ -0,0 +1,402 @@
...
+    <label for="email">Email <em>*</em></label>
+    <input id="email" maxlength="80" name="email" size="20" type="email" required /><br>
+
+    <label for="company">Company/Organization <em>*</em></label>
+    <input id="company" maxlength="40" name="company" size="20" type="text" required /><br>
+
+    <label for="title">Job Title <em>*</em></label>
+    <input id="title" maxlength="40" name="title" size="20" type="text" required /><br>
+
+    <label for="country_code">Country <em>*</em></label>
```
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply bedrock/firefox/templates/firefox/enterprise/signup.html]: Added `required`.

File: media/css/firefox/enterprise.scss [hunk 6]
```diff
@@ -256,3 +269,97 @@ $color-photon-blue: #0099db;
         }
     }
 }
+
+/* -------------------------------------------------------------------------- */
+// Sign up page
+
+.enterprise-head {
+    @include clearfix;
+    @include font-size-level3;
```
[the remark this answers is not in the document]
Comment (stephaniehobson) [reply media/css/firefox/enterprise.scss]: Feeling too short on time to do this.

**Pull request merged**